connectors/src/connectors/snowflake/lib/snowflake_api.ts

```diff
@@ -17,7 +17,15 @@ export const testConnection = async ({
 }: {
   credentials: SnowflakeCredentials;
 }): Promise<Result<string, Error>> => {
-  const connection = snowflake.createConnection(credentials);
+  const connection = snowflake.createConnection({
+    ...credentials,
+
+    // Use proxy to have all requests coming from the same IP.
+    proxyHost: "proxy-service.default.svc.clusterset.local",
```

Comment:
WDYT about having `proxyHost` as a nullable env var, and only setup proxy if non-null ? that way it's no-op in dev

Reply:
Good catch. I can totally do that indeed.